### DISCUSSION TITLE
Are data URLs supported for images in Markdown documents?

### DISCUSSION THREAD

**Discussion by philbo** [jgm/pandoc#10023] [Q&A] [answered]
I have a Markdown document with an image as a base64 encoded [data URL](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs) which I'd like to convert to a Word document. However the image isn't included, only the link text. Does Pandoc support this feature? I have tried running with `--embed-resources` but without success. There is a note in the [documentation](https://pandoc.org/MANUAL.html#option--embed-resources[) suggesting _some_ support - but maybe only for HTML documents.

Markdown input:
```
![image link](data:image/png;base64,iVBORw0KGgoAAA.....)
```

Word Doc output:
```
image link
```

**Comment by philbo** [comment, marked as answer]
The issue was running Pandoc with `--sandbox=true`. I'm guessing it's protecting against some attack vector from data URIs. Disabling sandbox mode worked as expected - tested with pandoc 2.19.2. The error wasn't visible as I was using the `pypandoc` wrapper

**Reply by jgm** [reply]
Correct - with `--sandbox`, pandoc's markdown reader can't do any IO, including web requests.
